ironfish/src/storage/database/database.ts

```diff
@@ -260,4 +297,22 @@ export abstract class Database implements IDatabase {
       throw error
     }
   }
+
+  private callerName(): string {
+    try {
+      throw new Error()
+    } catch (e) {
```

Comment:
I think instead of try/catch here we can just create a new error directly. It doesn't have to be thrown

Reply:
updated